src/Chapter2.hs

```diff
@@ -336,7 +351,11 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList c d ef
+    | c < 0 = []
+    | d < 0 = []
+    | otherwise = (take ((d - c) + 1) (drop c ef))
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you can implement it more efficiently by adding a separate case to your check :slightly_smiling_face: